.github/CODEOWNERS

```
@@ -1 +1,2 @@
-*   @dotNomad @kgartland-rstudio @mmarchetti @sagerb @tdstein
+*   @dotNomad @kgartland-rstudio @mmarchetti @sagerb
+internal @tdstein
```

Just double checking does this need the leading or ending `/`?

```
# In this example, @doctocat owns any files in the build/logs
# directory at the root of the repository and any of its
# subdirectories.
/build/logs/ @doctocat
```

```
# In this example, @doctocat owns any file in the `/docs`
# directory in the root of your repository and any of its
# subdirectories.
/docs/ @doctocat
```

---

Oh, I think you are right.

```suggestion
/internal/ @tdstein
```